draft-rescorla-tls-ctls.md

```diff
@@ -66,6 +67,20 @@ as a related protocol or as a compression mechanism. Specifically, it
 can be implemented by a layer between the TLS handshake state
 machine and the record layer. See {{compression-layer}} for more details.
 
+## Protocol Embedding
+
+Most of the simplification in this protocol is focused on the TLS handshake,
+since that is where most of the complexity of TLS resides.  This compact
+handshake protocol should be usable in two different contexts: In an integrated
+protocol with the same application sematics as TLS, or as an independent
+Authenticated Key Exchange for use in other protocols (e.g., OSCORE).
+
+In the former case, handshake messages would be encapsulated in MLSPlaintext
+objects, as in TLS.  In the latter case, the sequences of handshake messages
+would be carried directly in the application protocol (as TLSCiphertext
+messages, when encrypted).  We assume that the application protocol will delimit
+messages as needed.
```

Comment:
I don't think I understand this text. Isn't it backwards? Why would they be TLSCiphertext in the second version